src/oic/oic/__init__.py

```diff
@@ -759,6 +759,9 @@ def user_info_request(self, method="GET", state="", scope="", **kwargs):
             else:
                 # use_authorization_header, token_in_message_body
                 if "use_authorization_header" in _behav:
+                    if _ttype.lower() == _ttype:
+                        # fix 'bearer' to 'Bearer' (standard requirement)
+                        _ttype = _ttype.capitalize()
                     token_header = "{type} {token}".format(type=_ttype,
```

Comment:
I think we should go by the old Postel 'rule': be liberal in what you accept from others.
So, 'bearer', 'BEARER', 'Bearer'  and so on all should be accepted.